packages/fxa-content-server/app/scripts/templates/support.mustache

```diff
@@ -25,17 +25,10 @@
     </header>
     <div class="child-views">
       <div class="settings-child-view support">
-        <div class="settings-unit">
-          <div class="settings-unit-stub">
-            <header class="settings-unit-summary">
-              <h2 class="settings-unit-title">Subscriptions</h2>
-            </header>
-          </div>
-        </div>
         <div class="settings-unit">
           <div class="support-form">
             <header>
-              <h3>{{#t}}Contact Us{{/t}}</h3>
+              <h2>{{#t}}Contact Us{{/t}}</h2>
```

Comment:
I think this should remain an `h3` as it makes sense for it to fall under "Subscriptions". (It looks like we have an empty `h2` rendered if there's no display name, I may not have latest though)

![image](https://user-images.githubusercontent.com/13018240/67016229-77266180-f0bd-11e9-817b-e328f56b35ec.png)

Reply:
sure!